fvtest/porttest/omrstrTest.cpp

```diff
@@ -1279,11 +1276,11 @@ TEST(PortStrTest, str_convRoundTrip)
 	uint32_t unicodeEnd = 0x10000; /* overestimate */
 	uint32_t numCodePoints = 0;
 	size_t bufferSize = (6 * unicodeEnd) + 2; /* loose upper bound */
-	U_16 *unicodeData = omrmem_allocate_memory(unicodeEnd * 2, OMRMEM_CATEGORY_PORT_LIBRARY);
-	uint8_t *mutf8Buffer = omrmem_allocate_memory(bufferSize, OMRMEM_CATEGORY_PORT_LIBRARY);
-	uint8_t *unicodeResult = omrmem_allocate_memory(bufferSize, OMRMEM_CATEGORY_PORT_LIBRARY);
+	uint16_t *unicodeData   = (uint16_t *) omrmem_allocate_memory(unicodeEnd * 2, OMRMEM_CATEGORY_PORT_LIBRARY);
+	uint8_t  *mutf8Buffer   = (uint8_t *)  omrmem_allocate_memory(bufferSize, OMRMEM_CATEGORY_PORT_LIBRARY);
+	uint8_t  *unicodeResult = (uint8_t *)  omrmem_allocate_memory(bufferSize, OMRMEM_CATEGORY_PORT_LIBRARY);
```

Comment:
Please don't align the variable names, ='s, and omrmem_allocate_memory()s.  Remove the padding spaces.